Divine/CLI/CommandLineArguments.cs

```diff
@@ -0,0 +1,310 @@
+                    break;
+                case "lsj":
+                    resourceFormat = ResourceFormat.LSJ;
+                    break;
+                case "lsx":
+                    resourceFormat = ResourceFormat.LSX;
+                    break;
+            }
+
+            return resourceFormat;
+        }
+
+        public static PackageVersion GetPackageVersion(string versionOption)
+        {
+            PackageVersion version;
+            
+            switch (versionOption)
+            {
+                case "v7":
+                    version = Enums.PackageVersion.v7;
+                    break;
+
+                case "v9":
+                    version = Enums.PackageVersion.v9;
+                    break;
+
+                case "v10":
+                    version = Enums.PackageVersion.v10;
+                    break;
+
+                case "v13":
+                    version = Enums.PackageVersion.v13;
+                    break;
+
+                default:
+                    version = Enums.PackageVersion.v13;
+                    break;
+            }
+
+            return version;
+        }
+
+        public static Dictionary<string, object> GetCompressionOptions(string compressionOption, PackageVersion version)
+        {
+            CompressionMethod compression;
+            bool fastCompression;
+
+            switch (compressionOption)
+            {
+                case "zlibfast":
+                    compression = LSLib.LS.CompressionMethod.Zlib;
+                    fastCompression = true;
+                    break;
+
+                case "zlib":
+                    compression = LSLib.LS.CompressionMethod.Zlib;
+                    fastCompression = false;
+                    break;
+
+                case "lz4":
+                    compression = LSLib.LS.CompressionMethod.LZ4;
+                    fastCompression = true;
+                    break;
+
+                case "lz4hc":
+                    compression = LSLib.LS.CompressionMethod.LZ4;
+                    fastCompression = false;
+                    break;
+
+                default:
+                    compression = LSLib.LS.CompressionMethod.None;
+                    fastCompression = false;
+                    break;
+            }
+
+            // fallback to zlib, if the package version doesn't support lz4
+            if (compression == LSLib.LS.CompressionMethod.LZ4 && version <= (PackageVersion) 9)
+            {
+                compression = LSLib.LS.CompressionMethod.Zlib;
+                fastCompression = false;
+            }
+
+            var compressionOptions = new Dictionary<string, object>
+            {
+                { "Compression", compression },
+                { "FastCompression", fastCompression }
+            };
+
+            return compressionOptions;
+        }
+
+        public static Dictionary<string, bool> GetGraphicsOptions(string[] options)

```

Comment:
See the comment above regarding "graphics".
(I won't copy-paste the same message to other occurrences, just assume that everything that uses the name "Graphics" for GR2/DAE is marked.)